CMakeLists.txt

```
@@ -100,6 +99,7 @@ include_directories(
 )
 
 add_library(ndt_omp
```

Using `ament_auto_add_library` doesn't work?

https://github.com/ament/ament_cmake/blob/a16e5cb7e2e2cd03bcabbe4a73348274e5fc5831/ament_cmake_auto/cmake/ament_auto_add_library.cmake#L82
https://github.com/ament/ament_cmake/blob/a16e5cb7e2e2cd03bcabbe4a73348274e5fc5831/ament_cmake_auto/cmake/ament_auto_package.cmake#L69-L77

https://github.com/ament/ament_cmake/blob/a16e5cb7e2e2cd03bcabbe4a73348274e5fc5831/ament_cmake_auto/cmake/ament_auto_add_library.cmake#L69-L71

```suggestion
ament_auto_add_library(ndt_omp
```

---

Ah, I had tried that once and it didn't quite work right because it only builds a static library, but it does not add `-fPIC` to the compiler flags, so that library can't be linked into other dynamic libraries.

After digging around a little bt in `ament_auto_add_library`, it looks like you can also add the `SHARED` option to make it build a shared library, and that works for me, so I'll switch it to use that.